app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
```

Comment:
It is a good practice to keep the `try` block as small as possible to prevent catching exceptions that you don't intend to handle. In this case, you can move the `for` loop inside the `try` block.